qt/scientific_interfaces/Indirect/Iqt.cpp

```diff
@@ -280,17 +280,23 @@ void Iqt::errorsClicked() {
 
 bool Iqt::isErrorsEnabled() { return m_uiForm.cbCalculateErrors->isChecked(); }
 
-double Iqt::getXMinValue(MatrixWorkspace_const_sptr workspace,
+/** 
+ * Gets the first x value which corresponds to a y value below 1
```

Comment:
Could this say why we need this as well? Why do we need the first X value? Is it used for plotting or cropping or in some other way?